pkgs/by-name/gt/gtuber/package.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  pkg-config,
+  gobject-introspection,
+  vala,
+  glib,
+  libsoup_3,
+  json-glib,
+  libxml2,
+  gst_all_1,
+}:
+
+stdenv.mkDerivation {
+  pname = "gtuber";
+  version = "0-unstable-2024-10-11";
+
+  src = fetchFromGitHub {
+    owner = "Rafostar";
+    repo = "gtuber";
+    rev = "468bf02a8adcf69b1bd6dd7b5dbcdcc0bfdb6922";
+    hash = "sha256-pEiHqcxkrxZRD9xW/R9DNDdp5foxaHK2SAuzmPNegaY=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    gobject-introspection # For g-ir-scanner
+    vala # For vapigen
+  ];
+  buildInputs = [
+    glib
+    libsoup_3
+    json-glib
+    libxml2
+    gst_all_1.gstreamer
+    gst_all_1.gst-plugins-base
+  ];
+
```

Comment:
set `passthru.updateScript` to `unstableGitUpdater` if we want to stay on latest commit all the time.